kibbeh/src/modules/layouts/DesktopLayout.tsx

```diff
@@ -22,37 +22,41 @@ export const DesktopLayout: React.FC<MainLayoutProps> = ({
   const screenType = useScreenType();
 
   let middle = null;
-
-  if (screenType === "3-cols") {
-    middle = (
+  
+  switch(screenType) {
```

Comment:
consistency - statements in project are padding with space around parenthesis - `switch (` not `switch(`

Reply:
alright so I replaced the indentation with spaces